Building nrf52840 example fails with: "unresolved import `digest::common::BlockSizes`"

Steps to reproduce:
1. Clone this lora-rs repository.
2. cd in to sub folder: examples/nrf52840
3. Execute: cargo build
4. Notice the build fails with: error[E0432]: unresolved import `digest::common::BlockSizes`

The error I get says that cmac could not be compiled:

>     Checking cmac v0.8.0-rc.4
> error[E0432]: unresolved import `digest::common::BlockSizes`
>   --> ~/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/cmac-0.8.0-rc.4/src/block_api.rs:11:14
>    |
> 11 |     common::{BlockSizes, InnerInit, InnerUser},
>    |              ^^^^^^^^^^
>    |              |
>    |              no `BlockSizes` in the root
>    |              help: a similar name exists in the module: `BlockSizeUser`
> 
> For more information about this error, try `rustc --explain E0432`.
> error: could not compile `cmac` (lib) due to 1 previous error
> warning: build failed, waiting for other jobs to finish...  

I am building on latest MacOS, using Rust 1.94 (released last week).

--- a/lorawan-encoding/Cargo.toml
+++ b/lorawan-encoding/Cargo.toml
@@ -15,7 +15,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
 aes = { version = "0.9.0-rc.4" }
-cmac = { version = "0.8.0-rc.4" }
+cmac = { version = "0.8.0-rc.5" }
 hex = { version = "0", default-features = false }
 defmt = { version = "0.3", optional = true }
 serde = { version = "1", default-features = false, features = [

